diff -Naur rustc-beta-src.orig/library/std/src/sys/unix/mod.rs rustc-beta-src/library/std/src/sys/unix/mod.rs
--- rustc-beta-src.orig/library/std/src/sys/unix/mod.rs	2021-01-10 21:29:54.758670454 +0100
+++ rustc-beta-src/library/std/src/sys/unix/mod.rs	2021-01-10 21:33:26.873636698 +0100
@@ -243,6 +243,9 @@
         #[link(name = "log")]
         #[link(name = "gcc")]
         extern "C" {}
+    } else if #[cfg(target_os = "linux")] {
+        #[link(name = "ssp_nonshared")]
+        extern "C" {}
     } else if #[cfg(target_os = "freebsd")] {
         #[link(name = "execinfo")]
         #[link(name = "pthread")]
